### PR TITLE
Added TTGO LORA v1.2 (T-FOX) compatability

### PR DESCRIPTION
**Pull request by jonashoechst**
[TTGO LORA v1.2 (T-FOX)](https://github.com/lewisxhe/TTGO-LoRa-Series) is a discontinued board, however it is still useful und used by our group.

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -157,6 +162,37 @@ lib_deps =  ${env.lib_deps} ${env.libs_gps} ${env.libs_ble}
 board_build.partitions = no_ota.csv
 
 
+; ================== T-BEAM ==================
```
Comment (gh0st42) [inline platformio.ini:165]: shouldn't these boards all be under a T-FOX section?

File: platformio.ini [hunk 2]
```diff
@@ -157,6 +162,37 @@ lib_deps =  ${env.lib_deps} ${env.libs_gps} ${env.libs_ble}
 board_build.partitions = no_ota.csv
 
 
+; ================== T-BEAM ================== 
+[env:ttgo_lora_v12]
```
Comment (gh0st42) [inline platformio.ini:166]: to stay in line with the other profiles the base name should be without display drivers and builds containing the display driver should have a `_display_` in their name to make it clear

File: platformio.ini (hunk 1 above)
Comment (gh0st42) [inline platformio.ini:165]: shouldn't these boards all be under a T-FOX section?

File: platformio.ini [hunk 3]
```diff
@@ -28,6 +28,11 @@ config_gps = -DUSE_GPS -DGPS_RX_PIN=34 -DGPS_TX_PIN=12
 config_lora = -DRFM95_CS=18 -DRFM95_RST=23 -DRFM95_INT=26
 config_gps = -DUSE_GPS -DGPS_RX_PIN=12 -DGPS_TX_PIN=15
 
+; TTGO LORA V1.2 (T-FOX)
+[ttgo_lora_v12]
```
Comment (gh0st42) [inline platformio.ini:32]: to not confuse them with the other lora 32 modules maybe adding `t-fox` similar to the `t-beam` might be a good idea

**Review by gh0st42 — CHANGES_REQUESTED** [review]

File: platformio.ini (hunk 3 above)
Comment (jonashoechst) [reply platformio.ini]: I've renamed the env's as well as the settings section in 97fdc0c, as requested.

File: platformio.ini (hunk 2 above)
Comment (jonashoechst) [reply platformio.ini]: I've appended `_display` to the names in 97fdc0c , as requested. 

However, introducing an `[env:*]` for every optional feature seems a quiet large overhead, thus I did not add non-display options for this board.

This discussion seems to be a bit out of scope in this PR, thus let's discuss it in #14.

**Pull request merged**